builder.py

```diff
@@ -31,6 +31,11 @@ def __init__(self):
     def new_building(self):
         self.building = Building()
 
+    def build_floor(self):
+        raise NotImplemented
```

Comment:
@MaximilianR Thank you. Fixed in new PR: https://github.com/faif/python-patterns/pull/113